packages/vuetify/src/components/VVirtualScroll/VVirtualScroll.tsx

```diff
@@ -72,11 +72,19 @@ export const VVirtualScroll = genericComponent<new <T, Renderless extends boolea
 
     useToggleScope(() => props.renderless, () => {
       onMounted(() => {
-        containerRef.value = getScrollParent(vm.vnode.el as HTMLElement, true)
-        containerRef.value?.addEventListener('scroll', handleScroll)
+        containerRef.value = getScrollParent(vm.vnode.el as HTMLElement, false)
```

Comment:
includeHidden needs to be true for places like VAutocomplete

Reply:
Rolled back. Tested it is unnecessary, must be a debugging code. Thanks for picking up